src/main/java/teammates/common/util/Const.java

```diff
@@ -253,14 +248,12 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String COURSE_REMIND = "Email an invitation to all students yet to join requesting them to join the course"
                 + " using their Google Accounts. Note: Students can use TEAMMATES without ‘joining’, but a joined student can access"
                 + " extra features e.g. set up a user profile";
-        public static final String COURSE_DELETE_ALL_STUDENTS = "Delete all students in this course";
 
         public static final String INSTRUCTOR_DISPLAYED_TO_STUDENT = "If this is unselected, the instructor will be completely invisible to students."
                 + " E.g. to give access to a colleague for ‘auditing’ your course";
         
         public static final String INSTRUCTOR_DISPLAYED_AS = "Specify the role of this instructor in this course as shown to the students";
     
-        public static final String STUDENT_COURSE_PROFILE = "Your profile in this course";
```

Comment:
made obselete in b082611...
~~~
commit b082611e5670253d5c3977f4f097bc693b9b9046
Author: Damith Rajapakse <damith@gmail.com>
Date:   Wed Jun 27 16:50:16 2012 +0800

    removed commented out teamforming tab, added Ui tests for access control for not-logged-in and unregistered users
~~~

added in 31ffaa8...
~~~
commit 31ffaa8eef9bac3f568cc6b255fea27b34c8c304
Author: aldrianobaja.m@gmail.com <aldrianobaja.m@gmail.com>
Date:   Thu Jun 14 20:55:56 2012 +0800

    Added Student Home page (with test)
    Added StudentEvalEdit page (without test) and its handler
    Cleanup more javascript
    Deleted some student page due to change in design
    Escape course name, evaluation name, and team name
    Changed method in Helper.java to make use of eval.getStatus
    Refactored some methods
    Temporary change in APIServlet
~~~